### PR TITLE
yamllint formatting cleanup for ocp-workload-starter-workshop

### PR DESCRIPTION
**Pull request by ritesh97-rh**
##### SUMMARY
For the specified roles:
- Create a yamllint file to enforce formatting rules
- Make changes to existing code base to adhere to formatting requirements
- yamllint errors will be displayed during formatting checks in future development

##### ISSUE TYPE
- Code Cleanup Request

##### COMPONENT NAME
ocp-workload-starter-workshop

### CODE REVIEW

File: ansible/roles/ocp-workload-starter-workshop/tasks/per_user.yml [hunk 1]
```diff
@@ -83,16 +83,20 @@
       tags: gogs
 
     - name: push catalog to git repository in Gogs for user "{{ my_user }}"
-      shell: |
-        git remote add {{ my_user }} http://{{ my_user }}:{{ user_gogs_password }}@{{ gogs_hostname }}/{{ my_user }}/{{ reponame_local }}.git
+      shell: >-
```
Comment (stencell) [inline ansible/roles/ocp-workload-starter-workshop/tasks/per_user.yml:86]: Why was this changed from literal to folded? It was not good before to be in a single `shell` task, but this will break it as is.

File: ansible/roles/ocp-workload-starter-workshop/tasks/remove_workload.yml [hunk 2]
```diff
@@ -15,14 +15,20 @@
 - name: Delete user projects
   command: "oc delete project {{ item }}"
   ignore_errors: true
-  with_sequence: start={{user_count_start}} end={{ user_count_end }} format={{ user_format }}
+  with_sequence:
```
Comment (stencell) [inline ansible/roles/ocp-workload-starter-workshop/tasks/remove_workload.yml:18]: Not certain it will work formatted like this. It will need to be tested.

**Review by stencell — CHANGES_REQUESTED** [review]
Testing and changes as noted.

File: ansible/roles/ocp-workload-starter-workshop/tasks/workload.yml [hunk 3]
```diff
@@ -63,7 +63,10 @@
   tags: gogs
 
 - include_tasks: per_user.yml
-  with_sequence: start={{user_count_start}} end={{ user_count_end }} format={{ user_format }}
+  with_sequence:
```
Comment (stencell) [inline ansible/roles/ocp-workload-starter-workshop/tasks/workload.yml:66]: This should not work.

File: ansible/roles/ocp-workload-starter-workshop/tasks/remove_workload.yml [hunk 4]
```diff
@@ -15,14 +15,20 @@
...
+    end: "{{ user_count_end }}"
+    format: "{{ user_format }}"
   tags: always
 
 # Remove users quota
 - name: Delete user quota
   command: oc delete clusterquota/clusterquota-{{ item }}
   ignore_errors: true
-  with_sequence: start={{user_count_start}} end={{ user_count_end }} format={{ user_format }}
+  with_sequence:
```
Comment (stencell) [inline ansible/roles/ocp-workload-starter-workshop/tasks/remove_workload.yml:28]: This should not work.

File: ansible/roles/ocp-workload-starter-workshop/tasks/per_user.yml (hunk 1 above)
Comment (ritesh97-rh) [reply ansible/roles/ocp-workload-starter-workshop/tasks/per_user.yml:86]: Line length constraints bc of the add url. Should not break .. double line in `>` converts to single new line. The yaml to json conversion looks the same. Could also separate the commands using `;` or `&&` as we would in shell.

Comment (stencell) [reply ansible/roles/ocp-workload-starter-workshop/tasks/per_user.yml:86]: but if you use folded, it will just string all of these lines together with a whitespace between them. These are all separate commands and, as you noted, this can be fixed with an `&&` option, but as of right now this would be broken.

Comment (ritesh97-rh) [reply ansible/roles/ocp-workload-starter-workshop/tasks/per_user.yml:86]: Double newline in folded = single newline in output. Single newline in `|` operator is the same as double newline in `>-` operator. That's why single newlines were converted to double newlines when the operator was changed from '|' to '>-`. It should not break.  The yaml to json conversion also looks the same.

Comment (stencell) [reply ansible/roles/ocp-workload-starter-workshop/tasks/per_user.yml:86]: Oh, you are right!

**Review by stencell — APPROVED** [review]
necessary changes made.
